src/game_api/render_api.cpp

```diff
@@ -153,6 +153,7 @@ void render_journal_page_place(JournalPage* page)
 }
 void render_journal_page_people(JournalPage* page)
 {
+    trigger_vanilla_render_journal_page_callbacks(ON::RENDER_PRE_JOURNAL_PAGE, JournalPageType::People, page);

```

Comment:
WIP or you missed that it's only for this one page?

Reply:
I was too perplexed both by it not existing yes and being so easy to add I forgot to do the rest.

Reply:
I think the main idea was: you can always draw on top of it with POST and most pages you can just make the elements invisible. 
Then I also added option to just make a blank page. So probably not many people had a need for it, but i don't hate it